marklogic-data-hub/src/test/java/com/marklogic/hub/deploy/commands/DeployExistingNamespacesTest.java

```diff
@@ -0,0 +1,122 @@
+package com.marklogic.hub.deploy.commands;
+
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.marklogic.hub.AbstractHubCoreTest;
+import com.marklogic.hub.DatabaseKind;
+import com.marklogic.mgmt.api.API;
+import com.marklogic.mgmt.api.database.Database;
+import com.marklogic.mgmt.resource.databases.DatabaseManager;
+import com.marklogic.mgmt.util.ObjectMapperFactory;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class DeployExistingNamespacesTest extends AbstractHubCoreTest {
```

Comment:
No need for a separate test here - just add the path-namespace stuff to the existing DeployDatabaseFieldCommandTest .